src/backend/utils/mmgr/mcxt.c

```diff
@@ -897,6 +950,15 @@ MemoryContextStats(MemoryContext context)
 
     MemoryContextStats_recur(context, context, name, namebuf, namebufsize, nBlocks, nChunks,
     		currentAvailable, allAllocated, allFreed, maxHeld);
+
+#ifdef EXTRA_DYNAMIC_MEMORY_DEBUG
+	/* Check, if chunks_htable is not NULL after printing stats, and free it */
+	if (chunks_htable)
```

Comment:
Back to this code part, please prove we really need it or chunks_htable is already destroyed.